projects/lesson-2/runtime/src/substratekitties.rs

```diff
@@ -0,0 +1,98 @@
+use support::{decl_storage, decl_module, StorageValue, StorageMap, dispatch::Result};
+
+//TODO-002
+pub trait Trait: balances::Trait {}
+
+//TODO-001
+use system::ensure_signed;
+
+//TODO-003
+use codec::{Encode, Decode};
+
+//TODO-004
+use sr_primitives::traits::{Hash};
+
+//#[derive(Encode, Decode, Default, Clone, PartialEq)]
+//#[cfg_attr(feature = "std", derive(Debug))]
+//pub struct MyStruct<A, B> {
+//    some_number: u32,
+//    some_generic: A,
+//    some_other_generic: B,
+//}
+
+#[derive(Encode, Decode, Default, Clone, PartialEq)]
+#[cfg_attr(feature = "std", derive(Debug))]
+pub struct Kitty<Hash, Balance> {
+    id: Hash,
+    dna: Hash,
+    price: Balance,
+}
+
+decl_storage! {
+    trait Store for Module<T: Trait> as SubstratekittiesModule {
+        // Declare storage and getter functions here
+//        Value: u64;
+//        SomeValue get(some_value_getter): map u32 => u32;
+//
+//        MyValue: map T::AccountId => u32;
+//
+//        MyItem: map T::AccountId => MyStruct<T::Balance, T::Hash>;
+
+          Kitties get(kitty): map T::Hash => Kitty<T::Hash, T::Balance>;
```

Comment:
This storage item is not enough to meet the requirements, like kitty has owner, check the [detail requirements](https://github.com/SubstrateOrg/Team5/blob/master/PPT/%E7%AC%AC%E4%BA%8C%E8%AF%BE%20%E7%AD%94%E6%A1%88.pdf)